### ISSUE TITLE
Use the 74193 IC to count up on the 7-segment display

### ISSUE THREAD

**Issue by surajkadapa** [acmpesuecc/7_seg_display#2]
Wire the 74193 IC in Logisim to the IC given (7447) and make sure it counts up from 0 to 9. \
P.S. Datasheets are a great way to get started.

**Comment by Chatter-77** [comment]
Can i get assigned to this issue?

**Comment by surajkadapa** [comment]
@Chatter-77 Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by daedalus-ode** [comment]
hey, can i be assigned?

**Comment by surajkadapa** [comment]
@Chatter-77 please give update on time needed in 10 mins, else the issue will be reassigned.

**Comment by surajkadapa** [comment]
@Chatter-77 reassigning due to lack of response.

**Comment by ssether04** [comment]
can i get assigned if this is still open?

**Comment by daedalus-ode** [comment]
extra time of 1hr approved by maintainer

**Comment by Alley2717** [comment]
can i try

**Comment by surajkadapa** [comment]
@daedalus-ode any update?
